### ISSUE TITLE
Release 1.13.1 Compiler Error using Swift Package Manager

### ISSUE THREAD

**Issue by jleishman** [aws-amplify/amplify-ios#1364]
### Describe the bug

Using release 1.13.1 causes a compiler error when brought in using Swift Package Manager:

```
amplify-ios/AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Support/SQLiteResultError.swift:34:20: error: cannot find 'SQLITE_CONSTRAINT' in scope
        if code == SQLITE_CONSTRAINT {
```

### Steps To Reproduce

```swift
Steps to reproduce the behavior:
1. Create a new iOS project (just select one of the templates)
2. Add the amplify-ios library using Swift Package Manager
3. Build
4. See error
```

### Expected behavior

Compilation should be successful.

### Amplify Framework Version

1.13.1

### Amplify Categories

API

### Dependency manager

Swift PM

### Swift version

5.0

### CLI version

4.50.2

### Xcode version

12.5.1

### Is this a regression? (i.e. was this working before a version upgrade)

1.13.0 does not exhibit this behavior

### Device

iPhone 12 Pro

### iOS Version

iOS 14.5

### Specific to simulators

No

**Comment by lawmicha** [comment]
Hi @jleishman, sorry for the inconvenience caused, we're still investigating why isn't working for SPM, and have #1365 that should fix the problem. We'll release this as 1.13.2 and deprecate 1.13.1. If you are trying to use the changes from 1.13.1, feel free to switch over to the `fix/sqlite` branch for now or revert to 1.13.0 to continue your development

**Comment by jleishman** [comment]
@lawmicha any chance that branch (or at least the latest commit on it) won't disappear if and when the pull request is merged?

**Comment by lawmicha** [comment]
Good question, we have auto delete enabled so I’ll make sure to reopen the branch when merging

**Comment by lawmicha** [comment]
we have release the fix in 1.13.3 https://github.com/aws-amplify/amplify-ios/releases/tag/v1.13.3 please let us know if you see any other issues
